oci/table_oci_bastion_session.go

```diff
@@ -0,0 +1,237 @@
+			{
+				Name:        "bastion_public_host_key_info",
+				Description: "The public key of the bastion host. You can use this to verify that you're connecting to the correct bastion.",
+				Hydrate:     getBastionSession,
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "lifecycle_state",
+				Description: "The current state of the session.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "time_created",
+				Description: "Time that bastion was created.",
+				Type:        proto.ColumnType_TIMESTAMP,
+				Transform:   transform.FromField("TimeCreated.Time"),
+			},
+
+			// Standard Steampipe columns
+			{
+				Name:        "title",
+				Description: ColumnDescriptionTitle,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("DisplayName"),
+			},
+
+			// Standard OCI columns
+			{
+				Name:        "tenant_id",
+				Description: ColumnDescriptionTenant,
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     plugin.HydrateFunc(getTenantId).WithCache(),
+				Transform:   transform.FromValue(),
+			},
+		},
+	}
+}
+
+//// LIST FUNCTION
+
+func listBastionSessions(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	region := d.KeyColumnQuals["region"].GetStringValue()
+	equalQuals := d.KeyColumnQuals
+	bastionId := equalQuals["bastion_id"].GetStringValue()
+
+	// handle empty keyId, endpoint and region in list call
+	if bastionId == "" {
+		return nil, nil
+	}
+
+	// Create Session
+	session, err := bastionService(ctx, d, region)
+	if err != nil {
+		return nil, err
+	}
+
+	request := bastion.ListSessionsRequest{
+		BastionId: types.String(bastionId),
+		Limit:     types.Int(100),
+		RequestMetadata: common.RequestMetadata{
+			RetryPolicy: getDefaultRetryPolicy(d.Connection),
+		},
+	}
+
+	if equalQuals["display_name"] != nil {
+		request.DisplayName = types.String(equalQuals["display_name"].GetStringValue())
+	}
+
+	// Check for limit
+	limit := d.QueryContext.Limit
+	if d.QueryContext.Limit != nil {
+		if *limit < int64(*request.Limit) {
+			request.Limit = types.Int(int(*limit))
+		}
+	}
+
+	pagesLeft := true
+	for pagesLeft {
+		response, err := session.BastionClient.ListSessions(ctx, request)
+		if err != nil {
+			return nil, err
+		}
+
+		for _, bastionSession := range response.Items {
+			d.StreamListItem(ctx, bastionSession)
+
+			// Context can be cancelled due to manual cancellation or the limit has been hit
+			if d.QueryStatus.RowsRemaining(ctx) == 0 {
+				return nil, nil
+			}
+		}
+		if response.OpcNextPage != nil {
+			request.Page = response.OpcNextPage
+		} else {
+			pagesLeft = false
+		}
+	}
+
+	return nil, err
+}
+
+// // HYDRATE FUNCTION
+func getBastionSession(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	logger := plugin.Logger(ctx)
+	region := plugin.GetMatrixItem(ctx)[matrixKeyRegion].(string)
+	logger.Debug("getBastionSession", "OCI_REGION", region)
```

Comment:
```suggestion

```

Reply:
Hi @misraved, Sure I can make those changes. This PR was done before I created the table generator.

Also just FYI There's another PR that contains all the changes to the SQL formatting. It's #454